djgeojson/views.py

```diff
@@ -149,7 +150,8 @@ def get_queryset(self):
         # Won't trim point geometries to a boundary
         model_field = qs.model._meta.get_field(self.geometry_field)
         self.trim_to_boundary = (self.trim_to_boundary and
-                                 not isinstance(model_field, PointField))
+                                 not isinstance(model_field, PointField) and
+                                 Intersection)
```

Comment:
`Intersection is not None` would be easier to read I think

Reply:
Good point. Fix incoming.